blog/how-to-use-databases-in-the-docker-world.md

```diff
@@ -0,0 +1,173 @@
+# How to use databases in the Docker world
+
+We can run DB engines locally on our machine without install them using Docker containers. We're going to check 
+
+💡 Remember that the container's data is removed with the container itself, so, we're going to store the DB data into a specific Docker volume (`--volume`) to keep it in time.
+
+## MongoDB
+
+We can start a MongoDB instance with this simple command:
+
+```bash
+docker run --detach --name mongo \
+  --publish 27017:27017 \
+  --volume mongo:/data/db \
+  mongo
+```
+
+This starts a Docker container that runs the `mongod` process, and it listens the default MongoDB port `27017`. By default MongoDB uses the `test` database and only accepts connections from `localhost` without authentication.
+
+### The Docker run command
+
+Let's explain the `docker run` arguments:
+
+1. `--detach` runs the container in background, like a service.
+2. `--name` assigns a specific name to the created container.
+3. `--publish` forwards a specific port in `localhost` to the given container.
+4. `--volume` creates or reuse a volume and attach it to the given container in the specified path.
+
+By default MongoDB uses the `/data/db` directory to store the databases, remember to mount a Docker volume in that path.
+
+### A MongoDB root user
+
+You can specify a `root` user to the DB adding the following environment variables you the container
+
+```bash
+docker run --detach --name mongo \
+  -env MONGO_INITDB_ROOT_USERNAME=mongoadmin \
+  --env MONGO_INITDB_ROOT_PASSWORD=secret \
+  --publish 27017:27017 \
+  --volume mongo:/data/db \
+  mongo
+```
+
+For more variables check the `mongo` image page in [Docker hub](https://hub.docker.com/_/mongo).
+
+### Using the MongoDB shell
+
+You can connect via terminal using the `mongosh` command in the same container.
+
+```bash
+docker exec -it mongo mongosh
+```
+
+Use `--username` and `--password` if those values are required.
+
+### How to update the engine
+
+We can update a DB instance easily if we previously stored the DB data in a Docker volume ‼️. This is import, otherwise we'll lose data.
```

Comment:
```suggestion
We can update a DB instance easily if we have previously stored the DB data in a Docker volume ‼️. This is important, otherwise we'll lose data.
```